test/index.test.js

```diff
@@ -0,0 +1,39 @@
+var should = require('should');
+
+var Sequel = require('../sequel/index');
+
+var userSchema = {
+  users: {
+    identity: 'users',
+    tableName: 'users',
+    attributes: {
+      id: {
+        primaryKey: true,
```

Comment:
I added a test in 5b05e12 - it should work OK, since you have to declare it on each column and we check each column individually.